File: benchmark/workloads/ruby/tableLoop.rb
values = Array.new(1_000_000)
index = 0
while index < values.length
  values[index] = index + 1
  index += 1
end

total = 0
index = 0
while index < values.length
  total += values[index]
  index += 1
end
puts total

File: benchmark/workloads/csharp/tableLoop.cs
var values = new long[1_000_000];
for (var index = 0; index < values.Length; index++)
{
    values[index] = index + 1;
}

long total = 0;
for (var index = 0; index < values.Length; index++)
{
    total += values[index];
}
Console.WriteLine(total);
